src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -52,9 +57,41 @@ public BrowserFixture()
             }
         }
 
+        public IWebDriver Browser { get; }
+
+        public ILogs Logs { get; }
+
+        public IMessageSink DiagnosticsMessageSink { get; }
+
+        public static bool HostSupportsBrowserAutomation => string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("ASPNETCORE_BROWSER_AUTOMATION_DISABLED")) &&
+            (IsAppVeyor || (IsVSTS && RuntimeInformation.OSDescription.Contains("Microsoft Windows")) || OSSupportsEdge());
```

Comment:
Is this only meant to run on Windows?

Reply:
For now yes, although I've cleaned up all this crap